### PR TITLE
Ensure Unicode-compatible .meta files

### PR DESCRIPTION
**Pull request by gfetterman**

### CODE REVIEW

File: specification.md [hunk 1]
```diff
@@ -202,39 +202,38 @@ Event data are stored in CSV files. Simple event data should be
...
+  SI notation. May be an empty string for sampled data if the units are not
+  known. Event data must have units of "samples" (for a discrete timebase) or
+  "s" (for a continuous timebase); sampled data must not use these units. For
+  complex event data, this attribute must be an array, with each element of the
+  array indicating the units of the associated field in the data.
 
 The following attribute is only required for datasets with a discrete timebase:
 
 - **sampling\_rate:** A nonzero number indicating the sampling rate of the data,
-  in samples per second (Hz). Required for all datasets with a sampled timebase.
```
Comment (kylerbrown) [inline specification.md:232]: What's the logic for removing this?

File: specification.md [hunk 2]
```diff
@@ -202,39 +202,38 @@ Event data are stored in CSV files. Simple event data should be
 stored in a single column CSV, with each element in the array indicating the time of the
 event **relative to the start of the dataset**. The first line of the file must contain `start,`,
 indicating that the column contains the times of the event data. Event datasets can be
-distinguished from sampled datasets because the file is a plaintext CSV and `units` attribute must be
```
Comment (kylerbrown) [inline specification.md:205]: This is a hold-over from Arf, where this was the only way to distinguish between sampled and event data. I feel like we should keep this in to ensure back compatibility. Also, I added the filetype attribute, which I'm not yet using (it's an idea at this point) but if you like that approach we could discuss that here.

File: specification.md (hunk 1 above)
Comment (kylerbrown) [reply specification.md:232]: Redundant with L229?

**Review by kylerbrown — CHANGES_REQUESTED** [review]
Great work Graham, I really appreciate these changes. I've made a couple comments, take a look. Maybe let's talk about how bark should distinguish between sampled and event data, and how tightly these two data types are tied to the raw binary and csv formats.

File: specification.md (hunk 1 above)
Comment (gfetterman) [reply specification.md:232]: Yes, redundant.

File: specification.md (hunk 2 above)
Comment (gfetterman) [reply specification.md:205]: Sampled datasets have a 'units' attribute as well, which currently appears to be null. Keeping this distinction for backwards compatibility makes sense, but if so perhaps `bark:write_sampled` should be changed to disallow 'units' to be anything but None.

Comment (gfetterman) [reply specification.md:205]: Additionally, that the 'units' distinction serves this compatibility purpose is probably useful information to put in the spec.

**Comment by kylerbrown** [comment]
Hey @gfetterman , the usage of `units` was unclear, I've attempted to clarify. What do you think?

**Review by kylerbrown — APPROVED** [review]

**Pull request merged**